libraries/AP_LTM_Telem/AP_LTM_Telem.cpp

```diff
@@ -121,7 +121,7 @@ void AP_LTM_Telem::send_Sframe(void)
     const uint16_t amp = (uint16_t) roundf(current * 100.0f);                          // current sensor (expects value in hundredth of A)
 
     // airspeed in m/s if available and enabled - even if not used - otherwise send 0
-    const AP_Airspeed *aspeed = AP::ahrs().get_airspeed();
+    const AP_Airspeed *aspeed = AP::airspeed();
     uint8_t airspeed = 0; // airspeed sensor (m/s)
     if (aspeed && aspeed->enabled()) {
```

Comment:
needs explicit nullptr chk instead of nonzero

Reply:
As elsewhere, unrelated to my PR.  Not a bad change, just unrelated.